copy/copy.go

```diff
@@ -136,7 +135,7 @@ func Image(ctx *types.SystemContext, policyContext *signature.PolicyContext, des
 		if err != nil {
 			return fmt.Errorf("Error preparing to verify blob %s: %v", digest, err)
 		}
-		if err := dest.PutBlob(digest, digestingReader); err != nil {
+		if err := dest.PutBlob(digest, blobSize, digestingReader); err != nil {
```

Comment:
note to myself, PutBlob will probably receive the reader as its first argument and others following (cause the first one is the one really needed)